src/charts/Crypto.cpp

```diff
@@ -188,6 +200,23 @@ std::string Crypto::aesDecrypt(const std::string& in, const std::string& key) {
     return std::string((char *) plain.data());
 }
 
+std::string Crypto::getFileSha256(const std::string &utf8Path) {
+    std::ifstream ifs (utf8Path, std::ios::in|std::ios::binary|std::ios::ate);

```

Comment:
Sorry for noticing this only now: `fs::ifstream` should be used, otherwise it will fail when people install X-Plane in directories with UTF-8 characters that are outside their currently selected ANSI codepage - rare, but happens and is the whole reason why I had to add this workaround for Windows.